meson_options.txt

```
@@ -80,6 +80,11 @@ option(
   type : 'boolean',
   value : false,
   description : 'Enable/Disable HTML documentation')
+option(
+  'manpages',
+  type : 'boolean',
+  value : false,
```

Combined with the necessary install change, this changes the default for installing man pages.
Thus, maybe we should enable it by default?
```suggestion
  value : true,
```

---

As building the manual page now requires sphinx, didn't want to enable it by default. Distributions can enable it explicitly, if they need the manpage.

---

Ok. But we should add an explicit hint in the release notes about that later.